File: Python/1047.py
(h_inicio, m_inicio, h_fin, m_fin) = map(int, input().split(" "))
t_minutos_inicio = h_inicio * 60 + m_inicio
t_minutos_fin = h_fin * 60 + m_fin

t = t_minutos_fin - t_minutos_inicio if t_minutos_fin > t_minutos_inicio else 1440 - \
    t_minutos_inicio + t_minutos_fin
h = t // 60
m = t % 60

print("O JOGO DUROU %d HORA(S) E %d MINUTO(S)" % (h, m))

File: Javascript/1047.js
var input = `7 8 9 10`;
let [h_inicio, m_inicio, h_fin, m_fin] = input
  .split(/[\s]/)
  .map((x) => Number(x));
let t_minutos_inicio = 60 * h_inicio + m_inicio;
let t_minutos_fin = 60 * h_fin + m_fin;
let t =
  t_minutos_fin > t_minutos_inicio
    ? t_minutos_fin - t_minutos_inicio
    : 1440 - t_minutos_inicio + t_minutos_fin;
let h = Math.floor(t / 60);
let m = t % 60;

console.log("O JOGO DUROU " + h + " HORA(S) E " + m + " MINUTO(S)");
